Source/WebCore/page/Chrome.h

```diff
@@ -71,6 +72,7 @@ class IntRect;
 class LocalFrame;
 class NavigationAction;
 class Page;
+enum class PlatformEventModifier : uint8_t;
```

Comment:
This should be moved up near the other `enum class`.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/c362f4b3787e3a1ab3edc66da8512902bcfb5587